themes/tiddlywiki/vanilla/print.tid

```diff
@@ -0,0 +1,195 @@
+
+.tc-single-tiddler-window .tc-tiddler-frame .tc-tiddler-title-icon, .tc-single-tiddler-window .tc-tiddler-frame .tc-title {
+	display: none;
+}
+
+@media print {
+
+.tc-single-tiddler-window .tc-tiddler-frame .tc-tiddler-title {
+display: none;
+}
+
+}
+
+</$reveal>
+
+<$reveal state="$(printState)$!!show-subtitles" type="nomatch" text="yes">
+
+.tc-single-tiddler-window .tc-tiddler-frame .tc-subtitle {
+	display: none;
+}
+
+@media print {
+
+.tc-single-tiddler-window .tc-print-tiddler-list-view .tc-tiddler-frame .tc-subtitle {
+	display: none;
+}
+
+}
+
+</$reveal>
+
+<$reveal state="$(printState)$!!show-tags" type="nomatch" text="yes">
+
+.tc-single-tiddler-window .tc-print-tiddler-list-view .tc-tiddler-frame .tc-tags-wrapper {
+display: none;
+}
+
+@media print {
+
+.tc-single-tiddler-window .tc-print-tiddler-list-view .tc-tiddler-frame .tc-tags-wrapper {
+	display: none;
+}
+
+}
+
+</$reveal>
+
+<$reveal state="$(printState)$!!show-titles" type="match" text="yes">
+
+.tc-single-tiddler-window .tc-print-tiddler-list-view .tc-tiddler-frame .tc-tiddler-title-icon, .tc-single-tiddler-window .tc-print-tiddler-list-view .tc-tiddler-frame .tc-title {
+	display: inline;
+}
+
+@media print {
+
+.tc-single-tiddler-window .tc-print-tiddler-list-view .tc-tiddler-frame .tc-tiddler-title-icon, .tc-single-tiddler-window .tc-print-tiddler-list-view .tc-tiddler-frame .tc-title {
+	display: inline;
+}
+
+}
+
+</$reveal>
+
+<$reveal state="$(printState)$!!show-subtitles" type="match" text="yes">
+
+.tc-single-tiddler-window .tc-print-tiddler-list-view .tc-tiddler-frame .tc-subtitle {
+	display: inline;
+}
+
+@media print {
+
+.tc-single-tiddler-window .tc-print-tiddler-list-view .tc-tiddler-frame .tc-subtitle {
+	display: inline;
+}
+
+}
+
+</$reveal>
+
+<$reveal state="$(printState)$!!show-tags" type="match" text="yes">
+
+.tc-single-tiddler-window .tc-print-tiddler-list-view .tc-tiddler-frame .tc-tags-wrapper {
+	display: inline;
+}
+
+@media print {
+
+.tc-single-tiddler-window .tc-print-tiddler-list-view .tc-tiddler-frame .tc-tags-wrapper {
+	display: inline-block;
+}
+
+}
+
+</$reveal>
+
+\end
+
+.tc-print-tiddler-window-overlay p {
+	display: inline-block;
+	margin: 0;
+	-webkit-margin-before: 0em;
+	-webkit-margin-after: 0em;
+}
+
+.tc-print-tiddler-window-overlay button {
+	padding: .2em .5em 0em .5em;
+	font-size: 1.2em;
+	outline: none;
+}
+
+.tc-print-tiddler-window-overlay button.tc-print-tiddler-print-button {
+	padding-bottom: 0.1em;
+}
+
+.tc-print-tiddler-list-view .tc-story-river {
+	padding: 0;
+	width: calc(100vw - 2.05em);
+}
+
+.tc-print-tiddler-list-view .tc-tiddler-frame {
+	box-shadow: none;
+	border: none;
+	width: calc(100vw - 2.05em);
+}
+
+.tc-print-tiddler-list-view .tc-print-tiddler-button {
+	display: none;
+}
+
+.tc-print-sort-drop-down {
+	text-align: left;
+	-webkit-box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.5);
+	-moz-box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.5);
+	box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.5);
```

Comment:
I think `box-shadow` https://caniuse.com/#search=box-shadow doesn't need prefixing anymore. ... IMO we don't need shadows for printing. .. Can / Should we remove it?

Reply:
this box-shadow is for the "sort" dropdown in the small top print-toolbar only.
for the tiddler frames it's removed

Reply:
ah. ok